pkg/registry/extensions/controller/storage/storage.go

```diff
@@ -58,7 +60,7 @@ var _ = rest.Patcher(&ScaleREST{})
 
 // New creates a new Scale object
 func (r *ScaleREST) New() runtime.Object {
-	return &extensions.Scale{}
+	return &autoscaling.Scale{}
```

Comment:
@liggitt IMO we should not change any codes in `pkg/registry/extensions/controller/storage` directory as it is relevant to replication controller, which is v1.

Reply:
not an option, the internal `extensions.Scale` type is going away. This has no external API implications.

Reply:
sgtm :)

Reply:
Also, this is only used for `/apis/extensions/v1beta1/namespaces/ns/replicationcontrollers/rc/scale`, which is not how replication controllers should be scaled. The `/api/v1/namespaces/ns/replicationcontrollers/rc/scale` API is the v1 scale subresource for replication controllers

Reply:
noted! didn't know about that, thanks for pointing it out!